ReferenceApp/src/main/java/com/google/ads/interactivemedia/v3/samples/videoplayerapp/VideoListFragment.java

```diff
@@ -150,4 +124,31 @@ public void onResume() {
       resumeCallback.onVideoListFragmentResumed();
     }
   }
+
+  private String getRawFileContents(int resourceId) {
+    InputStream vastContentStream = getContext().getResources().openRawResource(resourceId);
+
+    StringBuilder stringBuilder = new StringBuilder();
+    BufferedReader reader = null;
+    try {
+      reader = new BufferedReader(new InputStreamReader(vastContentStream));
+
+      String line;
+      while ((line = reader.readLine()) != null) {
+        stringBuilder.append(line);
+      }
+    } catch (IOException e) {
+      e.printStackTrace();
```

Comment:
just rethrow. make the method have "throw IOException". swallowing exceptions like this is evil :-).

Reply:
I am surprised something like this is not built in by now

Reply:
I end up having to keep bubbling this up which is really annoying.  Although at the very least it makes more sense to catch it further up.

Reply:
the other standard trick is to wrap it in something that extends RuntimeException so that it is no longer checked

Reply:
Planning to leave the method as is (but fixing exceptions).  The approaches don't work as straightforward because of various version incompatibilities like with API and Java versions.